### PR TITLE
Extra logs

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: p2p/libp2p/discovery/optimizedKadDhtDiscoverer.go [hunk 1]
```diff
@@ -231,7 +238,7 @@ func (okdd *optimizedKadDhtDiscoverer) findPeers(ctx context.Context) {
 
 	err := okdd.kadDHT.Bootstrap(ctx)
 	if err != nil {
-		okdd.log.Debug("kad dht bootstrap", "error", err)
+		okdd.log.Debug("kad dht bootstrap", "error", err, "network", okdd.networkType)
```
Comment (iulianpascalau) [inline p2p/libp2p/discovery/optimizedKadDhtDiscoverer.go:241]: valid for all prints: can we put the network as the first parameter or in the log message for easier filtering when searching through the logs?

File: p2p/libp2p/netMessenger.go [hunk 2]
```diff
@@ -97,6 +98,7 @@ type ArgsNetworkMessenger struct {
 	P2pPrivateKey         commonCrypto.PrivateKey
 	P2pSingleSigner       commonCrypto.SingleSigner
 	P2pKeyGenerator       commonCrypto.KeyGenerator
+	NetworkType           p2p.NetworkType
```
Comment (iulianpascalau) [inline p2p/libp2p/netMessenger.go:101]: I think we should add the network on the 2 prints from this file:
```
network connection status ... 
network connection metrics ...
```

File: p2p/libp2p/discovery/optimizedKadDhtDiscoverer.go (hunk 1 above)
Comment (sstanculeanu) [reply p2p/libp2p/discovery/optimizedKadDhtDiscoverer.go:241]: updated

File: p2p/libp2p/netMessenger.go (hunk 2 above)
Comment (sstanculeanu) [reply p2p/libp2p/netMessenger.go:101]: already done, these logs are now part of connectionsHandler: https://github.com/multiversx/mx-chain-communication-go/pull/33/files#diff-d3c10daf4322b3b85154be491f3b8264cd5ad0d9b555b46f096e50b2c199dc1aR428

Comment (iulianpascalau) [reply p2p/libp2p/netMessenger.go:101]: oh, missed that

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**